src/components/NodeEditable.tsx

```diff
@@ -84,7 +111,9 @@ class NodeEditable extends Component<Props> {
         console.log(errorText);
         this.ignoreBlur = false;
         setErrorId(target.node ? target.node.id : "editing");
-        this.setSelection(false);
+        if (this.element.current) {
+          selectElement(this.element.current, false);
+        }
```

Comment:
this is where I was able to remove a call to `setAfterDOMUpdate`, which was hidden behind the `setSelection` method. There is no telling when this `onError` callback will fire, and it could be long after this component has been unmounted, which is why we check to see if our element still exists before selecting it.